changelog.md

```diff
@@ -1,3 +1,5 @@
+2024-11-08 - 30e71393cff4e116c842c56bf35ef84bd659448d - Update OXD CheckBox/SelectInput Component
```

Comment:
1 - Please follow the change log pattern

`{date}-{commit hash}-{file paths}-{meaningful description}`
```md
2024-11-08 - 30e71393cff4e116c842c56bf35ef84bd659448d - components/Input/CheckboxGroup.vue, components/Input/CheckboxInput.vue - Update OXD CheckBox/SelectInput Component <-- yes updated but why?
```
2 - Following file change is not tracked
**components/src/core/components/Icon/icons.ts**